examples/tree.bzl

```diff
@@ -0,0 +1,24 @@
+# Constants used
+
+INPUT_FILE = "/tmp/dependency.txt"
+OUTPUT_FILE = "/tmp/copy.txt"
+
+def _impl(ctx):
```

Comment:
This needs to move into `mybuilder.bzl`. We also need better names. `mybuilder_generator` would be better than `tree`.